addons/diagnostic/XEH_preInit.sqf

```diff
@@ -18,7 +18,8 @@ GVAR(projectileTrackedUnits) = [];
 
 ADDON = true;
 
-if (getMissionConfigValue ["EnableTargetDebug", 0] isEqualTo 1) then {
+private _cfg = configFile >> "EnableTargetDebug";
+if (1 isEqualTo ([getMissionConfigValue ["EnableTargetDebug", 0], getNumber _cfg] select isNumber _cfg)) then {
```

Comment:
Should be:
```sqf
getMissionConfigValue ["EnableTargetDebug", getNumber (configFile >> "EnableTargetDebug")] isEqualTo 1
```
imo. There is no reason for an addon to overwrite the mission if the mission sets the value explicitly.